src/structures/Webhook.js

```diff
@@ -61,17 +60,29 @@ class Webhook {
 
     /**
      * The owner of the webhook
-     * @type {User}
+     * @type {?User|object}
      */
-    if (data.user) this.owner = data.user;
+    if (data.user) {
+      if (!this.client.users) this.owner = data.user;
```

Comment:
This logic seems like it may be borked. Also, the JSDoc block needs to occur directly before the assignment.